tests/devices/qutrit_mixed/test_qutrit_mixed_measure.py

```diff
@@ -471,4 +471,118 @@ def test_variance_measurement(self, observable, ml_framework, two_qutrit_batched
+            t2 = qml.s_prod(coeffs[1], qml.prod(*(qml.GellMann(i, 5) for i in range(n_wires))))
+            H = t1 + t2
+
+        state = create_initial_state(range(n_wires), like=math.get_interface(scale))
+        for op in ops:
+            state = apply_operation(op, state)
+        return measure(qml.expval(H), state)
+
+    @staticmethod
+    def expected(scale, coeffs, n_wires=5, offset=0.1, like="numpy"):
+        phase = offset + scale * qml.math.asarray(range(n_wires), like=like)
+        cosines = math.cos(phase / 2) ** 2
+        sines = -math.sin(phase)
+        return coeffs[0] * qml.math.prod(cosines) + coeffs[1] * qml.math.prod(sines)
+
+    @pytest.mark.autograd
+    @pytest.mark.parametrize("convert_to_hamiltonian", (True, False))
+    @pytest.mark.parametrize(
+        "coeffs",
+        [
+            (qml.numpy.array(2.5), qml.numpy.array(6.2)),
+            (qml.numpy.array(2.5, requires_grad=False), qml.numpy.array(6.2, requires_grad=False)),
+        ],
+    )
+    def test_autograd_backprop(self, convert_to_hamiltonian, coeffs):
+        """Test that backpropagation derivatives work in autograd with hamiltonians and large sums."""
+        x = qml.numpy.array(self.x)
+        out = self.f(x, coeffs, convert_to_hamiltonian=convert_to_hamiltonian)
+        expected_out = self.expected(x, coeffs)
+        assert qml.math.allclose(out, expected_out)
+
+        g = qml.grad(self.f)(x, coeffs, convert_to_hamiltonian=convert_to_hamiltonian)
+        expected_g = qml.grad(self.expected)(x, coeffs)
+        assert qml.math.allclose(g, expected_g)
+
+    @pytest.mark.jax
+    @pytest.mark.parametrize("use_jit", (True, False))
+    @pytest.mark.parametrize("convert_to_hamiltonian", (True, False))
+    def test_jax_backprop(self, convert_to_hamiltonian, use_jit):
+        """Test that backpropagation derivatives work with jax with hamiltonians and large sums."""
+        import jax
+
+        jax.config.update("jax_enable_x64", True)
+
+        x = jax.numpy.array(self.x, dtype=jax.numpy.float64)
+        coeffs = (5.2, 6.7)
+        f = jax.jit(self.f, static_argnums=(1, 2, 3, 4)) if use_jit else self.f
+
+        out = f(x, coeffs, convert_to_hamiltonian=convert_to_hamiltonian)
+        expected_out = self.expected(x, coeffs)
+        assert qml.math.allclose(out, expected_out)
+
+        g = jax.grad(f)(x, coeffs, convert_to_hamiltonian=convert_to_hamiltonian)
+        expected_g = jax.grad(self.expected)(x, coeffs)
+        assert qml.math.allclose(g, expected_g)
+
+    @pytest.mark.torch
+    @pytest.mark.parametrize("convert_to_hamiltonian", (True, False))
+    def test_torch_backprop(self, convert_to_hamiltonian):
+        """Test that backpropagation derivatives work with torch with hamiltonians and large sums."""
+        import torch
+
+        coeffs = [
+            torch.tensor(9.2, requires_grad=False, dtype=torch.float64),
```

Comment:
Setting `grad=False` explicitly here made me think, what about cases where the coefficients themselves are differentiable parameters? This is a less common but still valid use-case.

Reply:
@Gabriel-Bottrill this comment is resolved now, right?